v22.2/alter-backup-schedule.md

```diff
@@ -0,0 +1,190 @@
+- `first_run = 'now'`: Take the first full backup immediately rather than wait for its next `RECURRING` time.
+- `ignore_existing_backups`: Ignore any existing backups already present in the storage location.
+
+The output shows that the [`detached` option](#detached) is implicitly added to scheduled backups:
+
+~~~
+     schedule_id     |   label     | status |           first_run           | schedule |                    backup_stmt
+---------------------+-------------+--------+-------------------------------+----------+----------------------------------------------------
+  814155335856521217 | aws_backups | ACTIVE | 2022-11-15 16:48:10.667767+00 | @daily   | BACKUP INTO 'external://s3_storage' WITH detached
+(1 row)
+~~~
+
+### Change the storage location for scheduled backups
+
+You can change the storage location to which your backup schedule is taking backups with the `SET INTO` command. Use the schedule ID to specify the schedule to modify and the new storage location URI. This statement also changes the schedule's label to match the change in backup location:  
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER BACKUP SCHEDULE 814155335856521217 SET INTO 'external://gcs_storage', SET LABEL gcs_backups;
+~~~
+~~~
+     schedule_id     |    label    | status |       first_run        | schedule |                    backup_stmt
+---------------------+-------------+--------+------------------------+----------+-----------------------------------------------------
+  814155335856521217 | gcs_backups | ACTIVE | 2022-11-16 00:00:00+00 | @daily   | BACKUP INTO 'external://gcs_storage' WITH detached
+(1 row)
+~~~
+
+{{site.data.alerts.callout_info}}
+[Incremental backups](take-full-and-incremental-backups.html#incremental-backups) require a full backup in the storage location. Therefore, when you change the storage location for a backup schedule, CockroachDB will pause any scheduled incremental backups until the next full backup runs on its regular schedule cadence. Consider that if you change the storage location and then adjust the frequency of your scheduled backups before the next full backup, any newly added incremental backups will not be part of the pause after a storage location change. This could result in a reported error state for the incremental backups, which will not resolve until the next scheduled full backup.
+{{site.data.alerts.end}}
+
+### Adjust frequency of scheduled backups
+
+To adjust the frequency of your scheduled backups, use `SET` with `FULL BACKUP` and `RECURRING` for full and incremental backups. You can either define the frequency as a `STRING` or in [crontab](https://en.wikipedia.org/wiki/Cron) format. See the [Parameters](#parameters) table for more detail.
+
+The following command adds incremental backups to the schedule occurring hourly:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER BACKUP SCHEDULE 814155335856521217 SET FULL BACKUP '@daily', SET RECURRING '@hourly';
+~~~
```

Comment:
Unrelated: ignorable: this is a good example of the limitations of our `sql` highlighter, where `FULL BACKUP` should be treated as a valid SQL statement target, but only `SET` and `FULL` are, as reserved SQL words themselves. Something for another time. One day, I envision `~~~ crdbsql`. Anyways, no action here!

Reply:
Yeah, this comes up in `ALTER CHANGEFEED` too. Yesssss, `crdbsql` sounds magical :)